__tests__/index.js

```diff
@@ -5,7 +5,7 @@ const puppeteer = require('puppeteer')
 const handler = require('serve-handler')
 const http = require('http')
 const rmfr = require('rmfr')
-const renderToString = require('../render-to-string')
+const renderToString = require('../render-to-string').default
```

Comment:
why is this necessary now?

Reply:
> why is this necessary now?

CommonJS (Node's require() and exports) is not compatible with ES6 default exports. But in this case as we are talking about puppeteer tests I don't see a reason. We are not using TS here.

Reply:
> CommonJS (Node's require() and exports) is not compatible with ES6 default exports

Yes, that's the reason. Without it'll throw `TypeError: renderToString is not a function`

Reply:
We can also convert all the test files to TS but I don't have enough time for it.